### PR TITLE
Seam creation

### PR DESCRIPTION
**Pull request by sarastavnitser**

### CODE REVIEW

**Comment by sarastavnitser** [comment]
trying to figure out why I have to press "resize" multiple times for the image to actually reach the size of the frame. Also I'm trying to figure out why I have to press the "reset" button twice to set the frame back to the size of the original image

File: src/main/java/ArrayInverter.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        Color[][] newArray = new Color[cols][rows];
+        for (int i = 0; i < cols; i++) {
+            for (int j = 0; j < rows; j++) {
+                newArray[i][j] = array[j][i];
+            }
+        }
+        return newArray;
+    }
+
+    public static void main(String[] args) {
```
Comment (andrewoid) [inline src/main/java/ArrayInverter.java]: This should be a unit test.

File: src/main/java/HorizontalSeamCreator.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    public HorizontalSeamCreator() {
+    }
+
+    public int[] createSmallestHorizontalSeam(int[][] energyGrid) {
+        ArrayInverter arrayInverter = new ArrayInverter();
+        int[][] invertedEnergyGrid = arrayInverter.invert2DIntArray(energyGrid);
+        int height = invertedEnergyGrid.length;
+        int width = invertedEnergyGrid[0].length;
+        PointsAndPositions pointsAndPositions = new PointsAndPositions(invertedEnergyGrid, height, width);
```
Comment (andrewoid) [inline src/main/java/HorizontalSeamCreator.java:11]: What are points and positions?

**Comment by andrewoid** [comment]
You should not be adding .DS_Store to git.
Also you're adding .class files, which makes me believe there's something very wrong with how your IntelliJ/gradle. is configured.

**Review by andrewoid — CHANGES_REQUESTED** [review]

File: src/main/java/HorizontalSeamCreator.java (hunk 2 above)
Comment (sarastavnitser) [reply src/main/java/HorizontalSeamCreator.java:11]: Points is the array of the points of the shortest path from each position at the current row the program is at while creating a path. Positions is the 2d array that holds the moves for the path that can be traced back to recreate the shortest path. Does this need to be changed? Thank you!

**Pull request merged**